### PR TITLE
(maint) Update CHANGELOG for 1.2.1 release

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

**Comment by camlow325** [comment]
@justinstoller To help knockout the work for [TK-443](https://tickets.puppetlabs.com/browse/TK-443), should we do a release of clj-http-client and roll that up into this next clj-parent release as well?

**Comment by justinstoller** [comment]
Good call!

**Comment by justinstoller** [comment]
I'll update the typo and add an entry for clj-http-client once https://github.com/puppetlabs/clj-parent/pull/71 is merged.

**Comment by justinstoller** [comment]
Updated the ol' chang-a-lang-a-log.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## [1.2.1] - 2017-06-14
+
+- Update http-metrics to version 9.9.0
```
Comment (camlow325) [inline CHANGELOG.md]: Should this be `puppetlabs/http-client`, and to version `0.9.0`?

**Comment by justinstoller** [comment]
I give up

**Pull request merged**